Documentation/CRDs/Object-Storage/ceph-object-store-user-crd.md

```diff
@@ -60,3 +60,88 @@ spec:
     * `user-policy`
     * `odic-provider`
     * `ratelimit`
+
+### CephObjectStoreUser Reference Secret
+
+A Kubernetes secret is created by Rook with the name, `rook-ceph-object-user-<store-name>-<user-name>` in the same namespace of cephobjectUser, where:
+
+* `store-name`: The object store name in which the user will be created. This matches the name of the objectstore CRD.
+* `user-name`: The metadata name of the cephObjectStoreUser
+
+Application pods can use the secret to configure access to the object store.
+
+```yaml
+apiVersion: v1
+data:
+  AccessKey: ***       # [1]
+  Credentials: ***     # [2]
+  Endpoint: ***        # [3]
+  SecretKey: ***       # [4]
+kind: Secret
+...
+...
+type: kubernetes.io/rook
+
+```
+
+1. `AccessKey`: User access key for ceph S3.
+2. `SecretKey`: User secret key for ceph S3.
+3. `Credentials`: Comma separated values of all access and secret keys.
+4. `Endpoint`: Endpoint for ceph S3.
+
+#### User-Specified Reference Secret
+
+AccessKey, SecretKey, and Credentials can be specified before CephObjectStore creation or modified afterwards to support user-specified keys and key rotation. Endpoint cannot be overridden by users and will always be updated by Rook to match the latest CephObjectStore information.
```

Comment:
As mentioned, I am concerned about how users reading the doc may misconstrue the "user" in "user-specified" with the "user" in "CephObjectStoreUser." Let us try to find clearer wording for "user-specified" that doesn't have the potential to confuse readers.

```suggestion
AccessKey, SecretKey, and Credentials can be specified before CephObjectStore creation or modified afterwards to support custom access credentials and key rotation. Endpoint cannot be overridden by users and will always be updated by Rook to match the latest CephObjectStore information.
```